src/theme.js

```diff
@@ -28,6 +29,13 @@ const theme = createTheme({
     neutral: {
       main: '#64748B',
       contrastText: '#fff'
+    },
+    temporaryPurple: {
```

Comment:
I think you don't need to state that it's temporary. If we wish to change the color we can simply change it to purple.